packages/js/components/src/media-uploader/media-uploader.tsx

```diff
@@ -109,9 +110,10 @@ export const MediaUploader = ( {
 							// @ts-expect-error - TODO multiple also accepts string.
 							multiple={ multipleSelect }
 							render={ ( { open } ) =>
-								buttonText ? (
+								buttonText || buttonProps ? (
 									<Button
 										variant="secondary"
+										{ ...buttonProps }
```

Comment:
I think that passing down any property to the component could be not a good idea.

Also, since you're adding a new `button pops` property, you could move it down to the last position. With this, it could be possible to pass the `onClick` in case we want a custom handler.

Reply:
Done!